src/chatlist.rs

```diff
@@ -248,22 +234,15 @@ impl Chatlist {
                                   AND (hidden=0 OR state=?1)
                                   ORDER BY timestamp DESC, id DESC LIMIT 1)
                  WHERE c.id>9 AND c.id!=?2
-                   AND c.blocked=0
-                   AND NOT c.archived=?3
+                   AND (c.blocked==0 OR (c.blocked=2 AND NOT ?3))
```

Comment:
Why is this `==` while all others are `=`?

Reply:
Fixed.

It does not result in a different behavior though, just a style change.